syntax = "proto3";

package cafm.identity.project;

message ProjectAppService_projectByNameRequest {
  string name = 1;
}

message ProjectAppService_projectByNameResponse {
  string id = 1;
  string name = 2;
}

message ProjectAppService_projectsRequest {
  int32 resultFrom = 1;
  int32 resultSize = 2;
}

message ProjectAppService_projectsResponse {
  string response = 1; 
}


service ProjectAppService {
  rpc projectByName (ProjectAppService_projectByNameRequest) returns (ProjectAppService_projectByNameResponse) {}
  rpc projects (ProjectAppService_projectsRequest) returns (ProjectAppService_projectsResponse) {}
}